src/components/Search/Search.js

```diff
@@ -0,0 +1,105 @@
+import React, { useMemo, useState } from 'react';
+import Autosuggest from 'react-autosuggest';
+import {
+  FaSearch,
+  FaChevronCircleDown,
+  FaChevronCircleUp,
+} from 'react-icons/fa';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+
+import SearchService from './SearchService';
+import SearchToggle from './SearchToggle';
+
+import './Search.scss';
+
+function Search() {
+  const dispatch = useDispatch();
+  const activeTopic = useSelector(state => state.app.activeTopic);
+  const [suggestions, setSuggestions] = useState([]);
+  const [value, setValue] = useState('');
+  const { t } = useTranslation();
+
+  const searchService = useMemo(() => {
+    const clear = () => setSuggestions([]);
+    const upsert = (section, items) =>
+      setSuggestions(oldSuggestions => {
+        const index = oldSuggestions.findIndex(s => s.section === section);
+        const newSuggestions = [...oldSuggestions];
+        if (index === -1 && items) {
+          newSuggestions.push({ section, items });
+        } else if (items) {
+          newSuggestions[index] = { section, items };
+        }
+        return newSuggestions;
+      });
+    return new SearchService(activeTopic, clear, upsert, dispatch);
+  }, [activeTopic, setSuggestions, dispatch]);
+
+  return (
+    Object.keys(activeTopic.searches).length > 0 && (
+      <div className="wkp-search">
+        <SearchToggle>
```

Comment:
Do we need the searchToggle ?

Reply:
I tried to replicate the behavior from the old WKP. On small mobile screens the search input field is in conflict with the SBB logo, so I guess we need some kind of toggle. But I'm open to alternatives.

Reply:
Yes exactly the kind of behavior we fixed with autocomplete. It should work with just css :
https://plans2.dev.trafimage.geops.ch/bern?c=600120.07%2C199658.77&lang=de&layer=bern_aussenplan&x=600107.38&y=199654.45&z=19

Reply:
Good point, I'll try that.